relay/cmd/main.go

```diff
@@ -83,3 +96,42 @@ func RunRelay(ctx *cli.Context) error {
 
 	return nil
 }
+
+func buildICS(logger logging.Logger, config *Config) (core.IndexedChainState, error) {
+	rpcClient, err := rpc.Dial(config.EthClientConfig.RPCURLs[0])
+	if err != nil {
+		return nil, fmt.Errorf("failed to create rpc client: %w", err)
+	}
+
+	client, err := geth.NewMultiHomingClient(config.EthClientConfig, gethcommon.Address{}, logger)
+	if err != nil {
+		logger.Error("Cannot create chain.Client", "err", err)
+		return nil, err
+	}
+
+	tx, err := coreeth.NewWriter(logger, client, config.BLSOperatorStateRetrieverAddr, config.EigenDAServiceManagerAddr)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create eth writer: %w", err)
+	}
+
+	idx, err := coreindexer.CreateNewIndexer(
```

Comment:
Is this using a built-in indexer? Should we use a graph indexer instead?

Reply:
Converted to use a thegraph indexer